lib/ReactTags.js

```diff
@@ -251,18 +251,15 @@ class ReactTags extends Component {
     }
   }
 
-  handlePaste(e) {
-    if (!this.props.allowAdditionFromPaste) {
-      return;
-    }
-
-    e.preventDefault();
-
+  /**
+   * Used to determine how the pasted content is split.
+   * @returns {RegExp} Regular expression
+   */
+  get delimiterRegex() {
     // See: http://stackoverflow.com/a/6969486/1463681
     const escapeRegex = (str) =>
```

Comment:
this can be made an independent function in utils.js and also doesn't need to be an arrow function

Reply:
@jpbufe3  once you create utils.js file can you also add specs for that file in tests folder ? You can write the specs in this PR or create a new PR as well for the specs once this pr is merged.